src/jobs/test.yml

```diff
@@ -35,6 +35,18 @@ parameters:
       Optionally supply a custom package installation command, with any additional flags needed.
     type: string
     default: ""
+  test-results-path:
+    type: string
+    default: ""
+    description: |
+      If you use test-results-for, this will be configured automatically.
+      If you configure test results yourself (for example in your node config files) set this to the location that you write them to, and they will be uploaded.
+  test-results-for:
+    default: none
+    type: enum
+    enum: ["jest", "none"]
+    description: |
+      If this is set to jest, test results will automatically be produced and uploaded. You must have the package jest-junit in the dev dependencies of your project.
```

Comment:
I feel like this description could describe the overall purpose of the parameter a bit clearer:
"Some testing libraries may require additional dependencies to export result metadata as JUnit XML or Cucumber JSON. For Jest we recommend: https://github.com/jest-community/jest-junit"
Thoughts?

I also do wonder if a package like https://www.npmjs.com/package/jest-cucumber would be applicable here. Just a curiosity though 😄

Reply:
Hm, trying to think how to word this better. Right now orb test command assumes you are using jest-junit, e.g.:

```
steps:
        - run:
            name: Check for test reporter
            working_directory: <<parameters.app-dir>>
            command: npm list jest-junit || (echo "Add the package jest-junit to the dev dependencies of your project with `npm install --save-dev jest-junit`" && exit 1)
        - run:
            name: Run NPM Tests
            working_directory: <<parameters.app-dir>>
            command: npm run <<parameters.run-command>> -- --reporters=default --reporters=jest-junit
``` 

How about something like:
```
description: |
      Testing framework your project uses. If this is set to jest, test results will be automatically produced using [jest-junit]((https://github.com/jest-community/jest-junit) reporter and uploaded to CircleCI. You must have the package jest-junit in the dev dependencies of your project.
``` ? 

Also, does the description support markdown? :-)

Reply:
>  Right now orb test command assumes you are using jest-junit,

ah I see! Feels a bit along though. Not sure if this is any better but here's my spin on it:

```
description:  |
      Testing framework your project uses. 
      If this is set to jest, test results will be automatically produced. The jest-junit package is required as a dev dependency. See https://github.com/jest-community/jest-junit for more information.
```

> Also, does the description support markdown? :-)

Doesn't look like it is supported. :(